src/CarbonAware.WebApi/test/integrationTests/mocks/WattTimeDataSourceMocker.cs

```diff
@@ -73,4 +154,23 @@ public void Dispose()
     {
         _server.Dispose();
     }
+
+    private void SetupResponseGivenGetRequest(string path, string body, HttpStatusCode statusCode = HttpStatusCode.OK, string contentType = MediaTypeNames.Application.Json)
```

Comment:
Can be this simplified? Seems like only 2 params are used. If there is a need to enhance it in the future, that might come as part of that work, but not here imho.

Reply:
👍 We know for sure that mocking the status code is coming, so I would like to leave that for usability.  But I agree that we don't have a certain need to return non-JSON content.  I will remove that as a param until we need it